tests/acceptance/README.md

```diff
@@ -0,0 +1,24 @@
+Acceptance tests
+================
+
+The tests will run on a file generated by the reporter.
+The file is generated using Mugshot, so you first need to run a browser:
+
+```sh
+npm run browser
+```
+
+Then you can generate the report. Two tests are made to fail.
```

Comment:
Proper. We need tests for that 😛  We also need to make sure the chai plugin behaves correctly when negating the assertion.

We can leave all of that for a future issue and in the meantime adapt the testing script to not fail from the acceptance setup. It might be as simple as using `||` instead of `&&`.